ios/FluentUI.Demo/FluentUI.Demo/Resources/Assets.xcassets/CommandBar/Delete_24_Regular.imageset/Contents.json

```diff
@@ -0,0 +1,15 @@
+{
+  "images" : [
+    {
+      "filename" : "Delete_24_Regular.pdf",
```

Comment:
if possible we try to keep the fluent-system-icon file name as is so we can easily replace them later if there is a new icon for the same purpose